lib/datapackage.py

```diff
@@ -0,0 +1,335 @@
+                        add_val = input('Enter new value: ')
+                        obj[key][add_key] = add_val
+
+                    elif selection == '2':
+                        mod_key = input('Enter the key: ')
+                        if mod_key not in val:
+                            print("Invalid input! Key not found.")
+                            continue
+                        mod_val = input('Enter new value: ')
+                        obj[key][mod_key] = mod_val
+
+                    elif selection == '3':
+                        del_key = input('Enter key to be deleted: ')
+                        if del_key not in val:
+                            print("Invalid key: Not found")
+                            continue
+                        print("Removed "+str(del_key) + " : " + str(obj[key].pop(del_key)))
+
+                    elif selection == '4':
+                        do_remove = input("Are you sure (Yes, completely remove this entry)? (y/n): ")
+                        if do_remove.lower() in ['y', 'yes']:
+                            obj.pop(key)
+                            print("Removed "+key+" from script.\n")
+                        else:
+                            print("Aborted.")
+                            sleep(1)
+                    elif selection == '5' or selection == "":
+                        pass
+                    else:
+                        raise RuntimeError("Invalid input!")
+
+                elif isinstance(val,list):
+
+                    for i in range(len(val)):
+                        print(str(val[i]))
+                        if isinstance(val[i], dict):
+                            print("\n\t'"+key+"' has the following dict:\n"+str(val[i])+"\n")
+                            do_edit = input("Edit the dict in '"+key+"'? (y/N): ")
+
+                            if do_edit.lower() in ['y', 'yes']:
+                                edit_dict(obj[key][i], tabwidth+2)
+
+                    print("Select one of the following for the key '"+key+"': \n")
+                    print("1. Add an item")
+                    print("2. Delete an item")
+                    print("3. Remove from script")
+                    print("4. Continue (no changes)\n")
+
+                    selection = input("\nYour choice: ")
+
+                    if selection == '1':
+                        add_val = input('Enter new value: ')
+                        obj[key].append(add_val)
+
+                    elif selection == '2':
+                        del_val = input('Enter value to be deleted: ')
+                        if del_val not in obj[key]:
+                            print("Invalid value: Not found.")
+                            continue
+                        print("Removed "+str(obj[key].pop(del_key)))
+
+                    elif selection == '3':
+                        do_remove = input("Are you sure (Yes, completely remove this entry)? (y/n): ")
```

Comment:
like in the line [228](https://github.com/weecology/retriever/pull/596/files#diff-fe19ce9787a421223628d4355dc2f5e4R228)
personally it reads better if we say 
`input("Are you sure (Yes/y will, completely remove this entry)? (y/n): ")`
but we could do as you wish